src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,60 @@
+package duke.task;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+
+/**
+ * Represents a event type task.
+ */
+public class Event extends Task {
+
+
```

Comment:
Similarly, you may want to remove unnecessary line breaks here to make it consistent with other classes